feathr-impl/src/main/scala/com/linkedin/feathr/offline/join/workflow/AnchoredFeatureJoinStep.scala

```diff
@@ -113,10 +113,13 @@ private[offline] class AnchoredFeatureJoinStep(
         val containsFeature: Seq[Boolean] = anchorDFMap.map(y => y._1.selectedFeatures.contains(x)).toSeq
         !containsFeature.contains(true)
       })
+      log.warn(s"Missing data for features ${missingFeatures.mkString}. Default values will be populated for this column.")
+      SuppressedExceptionHandlerUtils.missingDataSuppressedExceptionMsgs += missingFeatures.mkString
+      SuppressedExceptionHandlerUtils.missingFeatures ++= missingFeatures
```

Comment:
I am not sure, but do you think this should be done for a missing SWA feature as well? i.e. an expansion based on a missing SWA feature?

Reply:
Btw I think we are not adding missing Anchored Features in the exception message. See file: feathr-impl/src/main/scala/com/linkedin/feathr/offline/join/workflow/AnchoredFeatureJoinStep.scala

In PR: https://github.com/feathr-ai/feathr/pull/1158/files#diff-b845d91162c37a2ae8cf03ffac0b98af87c851fcce60b1f377278a82ae32372b

Reply:
Good catch, done.